lib/compilers/golang.ts

```diff
@@ -130,32 +152,37 @@ export class GolangCompiler extends BaseCompiler {
                 prevLine = lineMatch;
             }
 
-            args = this.replaceJump(func, funcCollisions[func], ins, args, usedLabels);
-            res.push(`\t${ins}${args}`);
+            if (func) {
+                args = this.replaceJump(func, funcCollisions[func], ins, args, usedLabels);
+                res.push(`\t${ins}${args}`);
+            }
             return res;
         });
 
         // Find unused pseudo-labels so they can be filtered out.
         const unusedLabels = _.mapObject(labels, (val, key) => !usedLabels[key]);
 
-        return _.chain(lines)
-            .flatten()
-            .compact()
+        return lines
```

Comment:
This can now be done because instead of returning null when not processing a line, we return an empty array, which gets flattened to nothing, so there's no need for compcat